manifests/charts/gateways/istio-egress/templates/poddisruptionbudget.yaml

```diff
@@ -12,7 +12,6 @@ metadata:
     install.operator.istio.io/owning-resource: {{ .Values.ownerName | default "unknown" }}
     operator.istio.io/component: "EgressGateways"
 spec:
-  minAvailable: 1
```

Comment:
Did you test the upgrade case, the PDB resources are not allowed to update, this change may not work in upgrade case, pls double check that.

Reply:
@morvencao Can you elaborate a bit more on `the PDB resources are not allowed to update, this change may not work in upgrade case` so that I can address the concern?

Reply:
Fresh install should be OK.
What if user reapply the istio control plane, or upgrade the istio control plane, will it be OK?

Reply:
@morvencao in upgrade case, the current state of the resources cannot have a conflict (both minAvailable and maxUnavailable co-exist, k8s would not allow it), therefore, the current state of the resources will be moved over with no issues regardless if the upgrade pass go through this code path or not. To answer your question, it will be OK.

Reply:
I have the same concern? How is this supposed to work, seems like the PDB will not do anything anymore and will break on upgrade?

Reply:
users can install with helm and kubectl, we cannot fix it only for Istioctl